index.html

```diff
@@ -1,5 +1,5 @@
 <body>
 
-<h1>Hello from forloopZambia</h1>
-<p>We are learning git and github </p>
+<b><h1>Hello from forloopZambia</h1><b>
+<i><p>We are learning git and github </p><i>
```

Comment:
It is always wise to put inline element inside block element, like the i should be inside the p tag rather than the other way.